### PR TITLE
Added nodemailer email script.

### PR DESCRIPTION
**Pull request by souravtecken**
1. mainMail.js with methods for general email, and password reset email.
2. mailCredentialsSample.s - authentication sample

### CODE REVIEW

**Comment by adityavinodk** [comment]
I shall check it out.

File: server/sendEmail/mainMail.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+		});
+	})  	
+}
+
+const sendPasswordResetMail = (userEmail, username, resetLink) => {
+	const mailOptions = {
+		from: auth.user,
+		to: userEmail,
+		subject: '[The Alcoding Club] Password Reset',
+		text: `Hello ${username}, please reset password here: ${resetLink}`
```
[the remark this answers is not in the document]
Comment (souravtecken) [reply server/sendEmail/mainMail.js]: But I've already added a generic sendMail function above it, what about that?

**Comment by parthvshah** [comment]
Before merging, make sure it's working on the server.

**Comment by adityavinodk** [comment]
Please rebase and squash commits into one and then send the PR.

**Pull request closed without merging**